po/pt.po

```diff
@@ -0,0 +1,707 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE paru'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the paru package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: paru VERSION\n"
+"Report-Msgid-Bugs-To: https://github.com/Morganamilo/paru\n"
+"POT-Creation-Date: 2021-06-28 03:24+0000\n"
+"PO-Revision-Date: 2021-06-28 19:03+0100\n"
+"Language: pt\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: João JCFB <soparajogosetestes@gmail.com>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 3.0\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/clean.rs:31
+msgid "Do you want to remove ALL AUR packages from cache?"
+msgstr "Deseja remover todos os pacotes AUR da cache?"
+
+#: src/clean.rs:33
+msgid "Do you want to remove all other AUR packages from cache?"
+msgstr "Deseja remover todos os outros pacotes AUR da cache?"
+
+#: src/clean.rs:40
+msgid "Clone Directory: {}"
+msgstr "Pasta Clone: {}"
+
+#: src/clean.rs:46
+msgid ""
+"\n"
+"Diff Directory: {}"
+msgstr ""
+"\n"
+"Pasta Diff: {}"
+
+#: src/clean.rs:48
+msgid "Do you want to remove all saved diffs?"
+msgstr "Deseja remover todas as diffs guardadas?"
+
+#: src/clean.rs:64
+msgid "can't open diff dir: {}"
+msgstr "não é possível abrir pasta diff: {}"
+
+#: src/clean.rs:75 src/clean.rs:133
+msgid "could not remove '{}'"
+msgstr "não foi possível remover '{}'"
+
+#: src/clean.rs:93
+msgid "can't open clone dir: {}"
+msgstr "não foi possível abrir pasta clone: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "opção {} exige um valor"
+
+#: src/command_line.rs:158 src/command_line.rs:327
+msgid "option {} does not allow a value"
+msgstr "opção {} não aceita um valor"
+
+#: src/command_line.rs:318
+msgid "unknown option --{}"
+msgstr "opção desconhecida --{}"
+
+#: src/command_line.rs:321
+msgid "unknown option -{}"
+msgstr "opção desconhecida -{}"
+
+#: src/completion.rs:27
+msgid "failed to open cache file '{}'"
+msgstr "erro ao abrir ficheiro de cache '{}'"
```

Comment:
See https://github.com/Morganamilo/paru/pull/435#discussion_r660053767

Reply:
Thx I didn't know that. Should be good now.